docs/troubleshooting.md

```diff
@@ -342,3 +342,85 @@ Units: bytes
 - `lineage_avg_string_len`: overall average length of the lineage description string
 - `std_dev`: standard deviation of the lineage description string length
 
+### Connection statistics
+
+Those metrics sample values regarding connections stored in the ConnectionTracker
+at every reporting interval (=scrape interval), and over a sliding time window.
+
+They can be configured using
+[environment variables](references.md#environment-variables)(`ROX_COLLECTOR_CONNECTION_STATS*`).
+
+Each metric is declined for both ingoing/outgoing direction, and private/public
+peer location. Corresponding labels are added to the reported values.
+
+#### Total number of known connections
+
+```
+Component: ConnectionTracker
+Prometheus names: rox_connections_total
+Units: count
+```
+
+This is the number of connections known to the ConnectionTracker during a reporting interval.
```

Comment:
The number of connections is recorded for each scrape interval within a time window. Quantile="0.5" for example means that in 50% of the scrape intervals x or greater number of connections were reported.